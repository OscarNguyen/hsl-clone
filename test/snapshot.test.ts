import { it, expect } from "vitest";

it("should have the correct title", () => {
  const result = "Main features";
  expect(result).toMatchSnapshot();
});
