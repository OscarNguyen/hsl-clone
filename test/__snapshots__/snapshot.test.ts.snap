// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should have the correct title 1`] = `"Main features"`;

exports[`should render the app 1`] = `"Main features"`;
